VictoriaAlbert/VictoriaAlbert/APIRequestManager.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  APIRequestManager.swift
+//  VictoriaAlbert
+//
+//  Created by Tong Lin on 11/10/16.
+//  Copyright © 2016 Tong Lin. All rights reserved.
+//
+
+import Foundation
+
+internal class APIRequestManager {
+    internal static let manager: APIRequestManager = APIRequestManager()
+    init() {}
+    
+    func getObjects(keyWords: String, callback: @escaping ((Data?)->Void)){
+        let url: URL = URL(string: "http://www.vam.ac.uk/api/json/museumobject/search?q=\(keyWords)")!
```

Comment:
By embedding URLs into your ```APIRequestManager``` you're forced to update it every time you use it. If you pass URLs to it, it can essentially be one method ```getData``` that takes a url string and returns ```Data?``` that you can either serialize or turn into a UIImage or anything.